### ISSUE TITLE
Database problems with the Codeigniter 4 Preset

### ISSUE THREAD

**Issue by murilomagalhaes** [kool-dev/kool#375]
**Describe the bug**

When creating a container with the Codeigniter preset and MySQL or MariaDB databases, it won't create the database or set up the user/password informed on the env file. 

**Kool version and environment**

kool version 1.14.0
PHP7.4/Codeigniter/MariaDB||MYSQL

**To Reproduce**

Simple create a new container, set up the user/password/database on the env file and try to connect to it 

**Expected behavior**

It was supposed to crate the user/password/database informed on the env file

**Additional context**

I've tried the Laravel preset, and it worked just fine.

**Comment by danielsuguimoto** [comment]
@murilomagalhaes thanks for the ticket! Can you share your `.env` and `docker-compose.yml` files?

**Comment by murilomagalhaes** [comment]
Hello, @danielsuguimoto ! Here it is!

![env](https://user-images.githubusercontent.com/50220430/134370134-65ab0c9a-9bab-46ba-b1d5-c3a6155a699d.png)

![docker-compose](https://user-images.githubusercontent.com/50220430/134370140-ed5ace52-5b9e-4016-b89b-900ee1c649f1.png)

Of course i could just edit the docker-compose.yaml and change the env variables with the information i need, like this for example:

```
 MYSQL_ROOT_PASSWORD: testPassword 
 MYSQL_DATABASE: testDb 
 MYSQL_USER: testUser 
 MYSQL_PASSWORD: testPassword
```

But since i didn't need to do that with the Laravel preset, i thought it should work the same way for Codeigniter

**Comment by danielsuguimoto** [comment]
@murilomagalhaes Environment variables names are not matching. On `docker-compose.yml` you're trying to get the value from `DB_` alike variables...and codeigniter sets the variables as `data.default.`.

For now...you can set these variables on codeigniter `.env` file like this, I guess...

```
...
DB_DATABASE=testDb
DB_USERNAME=testUser
DB_PASSWORD=testPassword

database.default.database="${DB_DATABASE}"
database.default.username="${DB_USERNAME}"
database.default.password="${DB_PASSWORD}"
...
```

Or you can try using `database.default.database`, for example, directly on `docker-compose`... e.g.:

```
...
environment:
  MYSQL_DATABASE: ${database.default.database}
...
```

Then run `kool restart --purge`. I'm not a codeigniter developer, so I'm not sure if it works the same way as laravel does, but let me know if the changes I suggested didn't help at all...

Codeigniter preset was recently added by the community, so it's still fresh and it might need some adjustments...either on documentation or in the preset itself.

**Comment by murilomagalhaes** [comment]
Thank you for your reply, @danielsuguimoto!

When i've tried to edit the docker-compose file like you mentioned, i get this error when running kool run setup:

**_ERROR: Invalid interpolation format for "environment" option in service "database": "${database.default.database}"
exit status 1_**

But adding this to the env file worked
```
DB_DATABASE=testDb
DB_USERNAME=testUser
DB_PASSWORD=testPassword
```

**Comment by danielsuguimoto** [comment]
@murilomagalhaes No problems! Nice to see that it worked! We'll improve codeigniter preset documentation as soon as we have the chance! Thanks!

**Comment by fabriciojs** [comment]
The doc update has been merged onto `master`, will be published upon next release.

**Comment by fabriciojs** [comment]
This has been fixed within our [latest release 1.15.0](https://github.com/kool-dev/kool/releases/tag/1.15.0).
